templates/download/server/manual.html

```diff
@@ -1,213 +1,382 @@
 {% extends "download/_base_download.html" %}
 
-{% block title %}Get Ubuntu Server | Download{% endblock %}
```

Comment:
Let's not remove " | Download" from the title, it might affect SEO (I adjusted the copy doc accordingly)